tests/__tests__/console/ConsoleTable_test.re

```diff
@@ -0,0 +1,428 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */;
+
+open TestFramework;
+open Console.ConsoleTable;
+
+describe("Console.table", ({test}) => {
+  test("Basic cell", ({expect}) => {
+    let actual =
+      Table.createElement(
+        ~columns=[{width: 4}],
+        ~children=[Row.createElement(~children=["Cell"], ())],
+        (),
+      );
+    let expected = "--------\n" ++ "| Cell |\n" ++ "--------";
+    expect.string(actual).toEqual(expected);
```

Comment:
minor, but I think 
```expect.string(actual).toEqualLines([
"--------",
"| Cell |,
"--------"
])
```
might be more readable

Reply:
I didn't realize this matcher existed. That's perfect!